next-app/src/lib/forms/Button.svelte

```diff
@@ -0,0 +1,14 @@
+<script>
+	export let danger = false
+</script>
+
+<!-- https://daisyui.com/components/button -->
+<button on:click class:danger class='btn btn-primary'>
+	<slot />
+</button>
+
+<style>
+.danger { @apply
+	btn-error;
+}
```

Comment:
Does the `btn-error` style override `btn-primary`?

Also, would we want `export let primary` in the future, and/or a few other colors-as-props?

Reply:
> Does the btn-error style override btn-primary?

Yes it does, you can see that demonstrated on the `Index.svelte` page if you just `npm run dev` and hit `http://localhost:3000`

> Also, would we want export let primary in the future, and/or a few other colors-as-props?

Potentially as needed, `secondary` or `accent` perhaps.